include/slate/BaseMatrix.hh

```diff
@@ -4078,55 +4077,54 @@ void BaseMatrix<scalar_t>::eraseLocalWorkspaceTile(int64_t i, int64_t j)
 
         for (int device = HostNum; device < this->num_devices(); ++device) {
             if (tile_node.existsOn( device )
-                    && tile_node[ device ].tile()->workspace()) {
-
-                this->tileErase( i, j, device );
+                && tile_node[ device ].tile()->workspace()) {
```

Comment:
Just a comment:
I know it is not stated in the code style but, here, I find it difficult to read when the `{` is at the end of the multi-line condition. There is no indentation difference with the content of the next line.

Reply:
There's a question about this in the style guide. Do you have a preference? Part of the problem is adopting a style that auto-formatters like uncrustify or clang-format support.